spaceship.zsh

```diff
@@ -738,6 +746,25 @@ spaceship_julia() {
     "$SPACESHIP_JULIA_SUFFIX"
 }
 
+# JAVA
+# Show current Java version
+spaceship_java() {
+  [[ $SPACESHIP_JAVA_SHOW == false ]] && return
+
+  _exists java || return
+
+  # Show Java version only for Java-related folders
+  [[ -f pom.xml ]] || return
```

Comment:
How about support for Gradle as well as Maven, so looking for a `*.gradle` too?

Reply:
This should give : `[[ -f pom.xml || -f build.gradle || -f build.gradle.kts ]]`

Also while this plugin could be activated on various java build tool file, I would recommend to write the necessary build tool section for their respective versions.